### PR TITLE
Initial commit of CAIP samples using penguins dataset

### PR DESCRIPTION
**Pull request by ivanmkc**
TODO: Add explanation and link to bug here.

### CODE REVIEW

File: penguins/tf-keras/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Getting started: Training and prediction with Keras in AI Platform
+
+This code uses [`tf.keras`](https://www.tensorflow.org/guide/keras) to train a
+classifier on the [Census Income Data
```
Comment (ivanmkc) [inline penguins/tf-keras/README.md:4]: TODO: Fix comment here

File: penguins/tf-keras/trainer/util.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+# Keras specific
+from tensorflow.keras.utils import to_categorical
+
+# Storage directory
+DATA_DIR = os.path.join(tempfile.gettempdir(), "census_data")
```
Comment (ivanmkc) [inline penguins/tf-keras/trainer/util.py:32]: TODO

File: penguins/tf-keras/trainer/util.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+# Download options.
+DATA_URL = "https://storage.googleapis.com/cloud-samples-data/ai-platform/penguins"
+TRAINING_FILE = "penguins.csv"
+EVAL_FILE = "penguins.csv"
+TRAINING_URL = "%s/%s" % (DATA_URL, TRAINING_FILE)
+EVAL_URL = "%s/%s" % (DATA_URL, EVAL_FILE)
+
+# These are the features in the dataset.
+# Dataset information: https://archive.ics.uci.edu/ml/datasets/census+income
```
Comment (ivanmkc) [inline penguins/tf-keras/trainer/util.py:42]: TODO

File: penguins/tf-keras/trainer/util.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+    "sex",
+]
+
+# This is the label (target) we want to predict.
+_LABEL_COLUMN = "species"
+
+# These are columns we will not use as features for training. There are many
+# reasons not to use certain attributes of data for training. Perhaps their
+# values are noisy or inconsistent, or perhaps they encode bias that we do not
+# want our model to learn. For a deep dive into the features of this Census
```
Comment (ivanmkc) [inline penguins/tf-keras/trainer/util.py:59]: TODO: Fix this

File: penguins/tf-keras/trainer/util.py [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+# Keras specific
+from tensorflow.keras.utils import to_categorical
+
+# Storage directory
+DATA_DIR = os.path.join(tempfile.gettempdir(), "census_data")
+
+# Download options.
+DATA_URL = "https://storage.googleapis.com/cloud-samples-data/ai-platform/penguins"
+TRAINING_FILE = "penguins.csv"
```
Comment (ivanmkc) [inline penguins/tf-keras/trainer/util.py:36]: TODO: Replace with train file

File: penguins/tf-keras/README.md (hunk 1 above)
Comment (ivanmkc) [reply penguins/tf-keras/README.md:4]: Also change the link.

File: penguins/tf-keras/trainer/util.py (hunk 2 above)
Comment (ivanmkc) [reply penguins/tf-keras/trainer/util.py:32]: Replace "census"

File: penguins/tf-keras/trainer/util.py (hunk 3 above)
Comment (ivanmkc) [reply penguins/tf-keras/trainer/util.py:42]: Replace link.

File: penguins/tf-keras/trainer/util.py (hunk 4 above)
Comment (ivanmkc) [reply penguins/tf-keras/trainer/util.py:59]: Remove if not relevant for penguins.

File: penguins/tf-keras/trainer/util.py [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+
+# Keras specific
+from tensorflow.keras.utils import to_categorical
+
+# Storage directory
+DATA_DIR = os.path.join(tempfile.gettempdir(), "penguins_data")
+
+# Download options.
+DATA_URL = "https://storage.googleapis.com/cloud-samples-data/ai-platform/penguins"
+TRAINING_FILE = "penguins.data.csv"
```
Comment (ivanmkc) [inline penguins/tf-keras/trainer/util.py:36]: small nit: is that a typical filepath?

I've usually seen stuff like `penguins_train.csv`

Comment (ivanmkc) [reply penguins/tf-keras/trainer/util.py:36]: @aribray Please remove gs://cloud-samples-data/ai-platform/penguins/penguins.csv if you're not using it anymore.

**Comment by sgreenberg** [comment]
Closed in favor of https://github.com/GoogleCloudPlatform/cloudml-samples/pull/500

**Pull request closed without merging**